react-native-reanimated.d.ts

```diff
@@ -64,13 +63,17 @@ declare module 'react-native-reanimated' {
     class AnimatedValue<T extends Value> extends AnimatedNode<T> {
       constructor(value?: T);
 
-      value: T;
-
       setValue(value: Adaptable<T>): void;
 
       interpolate(config: InterpolationConfig): AnimatedNode<number>;
     }
 
+    class SharedValue<T extends Value> extends AnimatedNode<T> {
```

Comment:
It should not extend AnimatedNode, since it has nothing to do with AniimatedNode.

For example, by extending it, you inherit setValue and initerpolate. But Shared Values doesn't have these.